guides/common/modules/ref_advanced-settings-in-the-job-wizard.adoc

```diff
@@ -43,6 +43,10 @@ Is used if effective user is different from the ssh user.
 Concurrency level::
 Defines the maximum number of jobs executed at once.
 This can prevent overload of system resources in a case of executing the job on a large number of hosts.
++
+If a batch size is set, {SmartProxies} run batches of concurrent remote execution jobs.
+For Ansible-based remote execution jobs, change the `foreman_ansible_proxy_batch_size` setting in the {ProjectWebUI} under *Administer* > *Settings* on the *Ansible* tab.
```

Comment:
not quite, `foreman_ansible_proxy_batch_size` is empty by default. Then `foreman_tasks_proxy_batch_size` will be taken also for ansible-based remote execution jobs. If you set `foreman_ansible_proxy_batch_size` to some value, then this one overrides the batch size for ansible-based jobs.